src/wallet/rpcwallet.cpp

```diff
@@ -3736,32 +3736,56 @@ UniValue getaddressinfo(const JSONRPCRequest& request)
     UniValue ret(UniValue::VOBJ);
     CTxDestination dest = DecodeDestination(request.params[0].get_str());
 
-    // Make sure the destination is valid
+    // Ensure the destination is valid.
```

Comment:
1d6884d4d4b541d0c04ef10da208ef657122aa1f

The code below is clear and simple, why add the obvious comment?

Reply:
Removed.